docker-compose.yml

```diff
@@ -28,45 +28,36 @@ services:
     environment:
       - EXERCISM_ENV=development
       - EXERCISM_DOCKER=true
-      - MYSQL_HOST=mysql
-      - MYSQL_PORT=3306
-      - MYSQL_USERNAME=root
-      - MYSQL_PASSWORD=password
       - OVERMIND_SOCKET=/usr/src/app/tmp/overmind.sock
     ports:
       - 3020:3020
       - 3035:3035
       - 3334:3334
     volumes:
       - ../v3-website:/usr/src/app
-      - /usr/src/app/node_modules
       - type: tmpfs
-        target: /usr/src/app/tmp
+        target: /usr/src/tmp
 
   tooling-orchestrator:
     build:
       context: ../tooling-orchestrator
       dockerfile: Dockerfile.dev
     ports:
       - 3021:3021
-    volumes:
-      - ../tooling-orchestrator:/usr/src/app
-    depends_on:
```

Comment:
We don't need any depends_on, and it makes it slower if we have them, as it waits for one to start before starting the next. Better just to get everything running asap I think.

Reply:
Yep, makes sense